packages/server/src/service/webhooks.ts

```diff
@@ -20,6 +20,8 @@ export type GCPLogLevel = 'INFO' | 'WARNING' | 'ERROR'
 // rich information about the two interfaces here. These STUB definitions
 // will help us work around this limitation.
 
+export type MailgunEventReason = 'generic' | 'bounce' | 'suppress-bounce'
```

Comment:
Just to be sure for myself, there are no other mailgun event reasons?

Reply:
Hard to know, the documentation doesn't say what these can be, so I added all the ones I could find in their examples: https://documentation.mailgun.com/en/latest/api-events.html?highlight=user%20variables#event-structure